app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt

```diff
@@ -166,11 +167,15 @@ class AdyenPaymentFragment : BasePageViewFragment(), AdyenPaymentView {
     get() = bindingCreditCardPreSelected?.fragmentIabErrorPreSelected?.genericErrorLayout?.errorVerifyCardButton
       ?: bindingCreditCardLayout?.fragmentIabError?.genericErrorLayout?.errorVerifyCardButton!!
   private val layout_support_logo: ImageView
-    get() = bindingCreditCardPreSelected?.fragmentIabErrorPreSelected?.genericErrorLayout?.layoutSupportLogo
-      ?: bindingCreditCardLayout?.fragmentIabError?.genericErrorLayout?.layoutSupportLogo!!
+    get() = bindingCreditCardLayout?.fragmentAdyenError?.layoutSupportLogo
+      ?: bindingCreditCardPreSelected?.fragmentAdyenErrorPreSelected?.layoutSupportLogo
+      ?: bindingCreditCardLayout?.fragmentIabError?.genericErrorLayout?.layoutSupportLogo
+      ?: bindingCreditCardPreSelected?.fragmentIabErrorPreSelected?.genericErrorLayout?.layoutSupportLogo!!
```

Comment:
Using "!!" should be avoided, if this value could be null, it will crash. Are you sure it is always non null? Don't you prefer use a empty state or something in the last case?

Reply:
From my tests it's never null, but it should never reach that line either way.

Reply:
Do note that the last 2 lines are the old code and it seems they never caused any null-related problems.